rosa_learning/deploying_application_workshop/learning-lab-overview.adoc

```diff
@@ -0,0 +1,301 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="learning-lab-overview"]
+= Workshop overview
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: learning-lab-overview
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 22-JAN-2024
+//Modified for HCP 15 October 2024
+
+== Introduction
+
+After successfully provisioning your cluster, follow this workshop to deploy an application on it to understand the concepts of deploying and operating container-based applications. 
+
+.Workshop objectives
+
+* Deploy a Node.js-based application by using Source-to-Image (S2I) and Kubernetes deployment objects
+* Set up a continuous delivery (CD) pipeline to automatically push source code changes
+* Experience self healing applications
+* Explore configuration management through ConfigMaps, secrets, and environment variables
+* Use persistent storage to share data across pod restarts
+* Explore networking in Kubernetes and applications
+* Familiarize yourself with ROSA and Kubernetes functionality
+* Automatically scale pods based on loads from the Horizontal Pod Autoscaler
```

Comment:
```suggestion
* Automatically scale pods based on loads from the Horizontal Pod Autoscaler(HPA)
```

Reply:
Also, I believe Kubernetes refers to as HorizontalPodAutoscaler; https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
But it looks like we don't in our docs?